<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IconFontTool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(255, 255, 255);
}

QLabel{
border:none;
	color: rgb(51, 51, 51);
}

QToolButton{
	border:none;
}

QScrollBar:vertical {      
	border:none;           
    background:transparent;  
    width:6px;  
    margin: 0px 0px 2px 0px;  
}  
  
QScrollBar::handle:vertical {  
    background: rgb(195, 195, 195);  
    min-height: 20px;  
    border-radius: 3px;  
}  
  
QScrollBar::handle:vertical:hover{  
    background:rgba(0,0,0,30%);  
}  
  
QScrollBar::add-line:vertical {  
    height: 0px;  
    subcontrol-position: bottom;  
    subcontrol-origin: margin;  
}  
  
QScrollBar::sub-line:vertical {  
    height: 0px;  
    subcontrol-position: top;  
    subcontrol-origin: margin;  
} </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">color: rgb(119, 119, 119);</string>
        </property>
        <property name="text">
         <string>字符集</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="margin">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true">#scrollArea {
border:1px solid #eee;
}</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>395</width>
           <height>496</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout"/>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">#frame {
border:1px solid #eee;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lb_preview_128">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>128</width>
         <height>128</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 238);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>200</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:1px solid #ccc;
background-color: rgb(255, 255, 255);
color: rgb(51, 51, 51);</string>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(119, 119, 119);</string>
       </property>
       <property name="text">
        <string>预览</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_preview_64">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>103</y>
         <width>64</width>
         <height>64</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 238);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lb_preview_32">
       <property name="geometry">
        <rect>
         <x>274</x>
         <y>134</y>
         <width>32</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 238);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lb_preview_16">
       <property name="geometry">
        <rect>
         <x>325</x>
         <y>148</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 238);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>170</y>
         <width>18</width>
         <height>12</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(136, 136, 136);</string>
       </property>
       <property name="text">
        <string>128</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>213</x>
         <y>170</y>
         <width>12</width>
         <height>12</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(136, 136, 136);</string>
       </property>
       <property name="text">
        <string>64</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>284</x>
         <y>170</y>
         <width>12</width>
         <height>12</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(136, 136, 136);</string>
       </property>
       <property name="text">
        <string>32</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>328</x>
         <y>170</y>
         <width>12</width>
         <height>12</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(136, 136, 136);</string>
       </property>
       <property name="text">
        <string>16</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>191</width>
         <height>142</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>字符集:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lb_family">
            <property name="text">
             <string>family</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>字符:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lb_char">
            <property name="text">
             <string>name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>图标大小(px):</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>120</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>圆角(px):</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_2">
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>16</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>图标占比(1-100%):</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_3">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>90</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>360</y>
         <width>231</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>18</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>前景色(RGB):</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="color" native="true">
            <property name="styleSheet">
             <string notr="true">border:1px solid #eee;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">color: rgb(102, 102, 102);</string>
              </property>
              <property name="text">
               <string>背景色(RGBA):</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="bgcolor" native="true">
            <property name="styleSheet">
             <string notr="true">border:1px solid #eee;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <action name="action1">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
